### DISCUSSION TITLE
how do you delete an item from a list?

### DISCUSSION THREAD

**Discussion by notno** [ThePrimeagen/harpoon#630] [Q&A]
is it possible?

**Comment by olxandr** [comment]
You can just use "dd" to remove from the list in the Harpoon's ui. Not sure how to do it using Telescope's ui though.

**Reply by asuka-405** [reply]
it works in telescope ui as well

but there is a bug, if u press esc after deleting an item and not return, the list persists

**Comment by notno** [comment]
It doesn't seem to work for me in the non telescope quickmenu either. It will appear to delete, but it's still in the list when I reopen it.

**Reply by olxandr** [reply]
@notno 

You also have to save the changes with :w iirc.

**Comment by musticide** [comment]
Hey! Any update on this feature?

**Reply by allreddv** [reply]
I replied earlier with a complicated function that worked, but then I found this online. 

{ '<Leader>mr', function() require('harpoon'):list():remove() end, desc = 'Remove Location' },

**Reply by musticide** [reply]
Thanks a bunch!
